docs/src/components/Fragment.tsx

```diff
@@ -35,9 +39,14 @@ const shouldRenderFragment = (
   }
 };
 
-export const Fragment = ({ children, platforms }: FragmentProps) => {
+export const Fragment = ({
+  children,
+  platforms,
+  useCommonJsContent,
+}: FragmentProps) => {
   const { query } = useRouter();
-  const { platform = 'react' } = query;
+  const framework = (query.platform as string) ?? 'react';
```

Comment:
Non-blocking, but maybe we should think about 'platform' and 'framework' here and how we kind of use them interchangeably.